posthog/src/main/java/com/posthog/PostHogInterface.kt

```diff
@@ -164,6 +164,16 @@ public interface PostHogInterface {
      */
     public fun distinctId(): String
 
+    /**
+     * Returns the [anonymousId] property
+     */
+    public fun anonymousId(): String
```

Comment:
This is also implementation detail since users don't know about `anonymousId`, but only about `distinctId`.
It should not be part of the public API